src/Microsoft.NET.Sdk.Functions.Generator/TypeUtility.cs

```diff
@@ -95,5 +99,70 @@ internal static Attribute GetResolvedAttribute(ParameterInfo parameter, Attribut
 
             return attribute;
         }
+
+        public static Attribute ToReflection(this CustomAttribute customAttribute)
```

Comment:
I wish we didn't have to do this, but we need to invoke the constructor in order to have the properties populated correctly.

Alternative to avoid reflection completely is to sniff the constructor parameter names and assume that, for example, a parameter named 'blobPath' maps to the property named `BlobPath`.

Reply:
That would be pretty fragile. I think we're ok with this for now. The cases where this would be a potential issue are significantly less prominent than the other cases addressed by the changes. Let's leave this and discuss other approaches, but not block on that.